docs/source/user_guide/box_model/index.rst

```diff
@@ -1,13 +1,61 @@
 Box model
 =========
+This section details the components of the box model, including reactions, mechanisms, conditions, and solutions. To use the MusicBox model, import::
+    
+    from acom_music_box import MusicBox, Conditions
 
-This section details the components of the box model, including reactions, mechanisms, conditions, and solutions.
+Using the previously defined in-code mechanism (see :ref:`Defining chemical species <species>`), a box model can now be created and initialized with it::
+    
+    box_model = MusicBox()
+    box_model.load_mechanism(mechanism)
 
-.. toctree::
-   :maxdepth: 1
+Initial and evolving conditions
+--------------------------------
+Use `Conditions` to set environmental parameters and species concentrations. Initial conditions define the environment that 
+the mechanism takes place in at the start of the simulation through the parameters temperature (K), pressure (Pa), and optionally 
+air density. Without an air density provided, the Ideal Gas Law is used to calculate this parameter. The initial concentrations of each
+Species is also included::
+    
+    box_model.initial_conditions = Conditions( temperature=298.15, pressure=101325.0, species_concentrations={"X": 3.75, "Y": 5.0, "Z": 2.5,})
 
-   reactions
-   Chemical mechanisms <mechanisms>
-   Initial and evolving conditions <conditions>
-   Configuration options <configurations>
+Evolving conditions change the environment of the mechanism at a defined time value (Seconds), its first argument. Like Initial Conditions,
+these changes can also include temprature, pressure, air density, and Species concentrations::
+    
+    box_model.add_evolving_condition(100.0, Conditions(temperature=310.0, pressure=100100.0))
 
+Additional model options
```

Comment:
for consistency purpose, suggest changing to `3. setting simulation length and time step size`